test/signalrclient-e2e-tests/connection_tests.cpp

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#pragma once
+
+#include "stdafx.h"
+#include <string>
+#include "cpprest\details\basic_types.h"
+#include "cpprest\json.h" 
+#include "connection.h"
+#include "hub_connection.h"
+#include "console_log_writer.h"
+
+using namespace signalr;
+using namespace utility;
+using namespace std;
```

Comment:
The convention I am using is not to use `using` unless namespaces are deeply nested. It makes it clear where the type comes from. Please follow this convention.